src/language-markdown/parsers.js

```diff
@@ -1,14 +1,16 @@
-"use strict";
+import markdownParser from "./parser-markdown.js";
```

Comment:
This became a top-level require in bundle, there is no way to make it work for both source and bundle. This will cause `index.js` slower to load (see #12262), I'll find a solution later.

A ESLint rule is disabled temporarily https://github.com/prettier/prettier/pull/12303/commits/839c531ad7554c81f15efdf2d56503dabdc5e45d